src/services/status.js

```diff
@@ -39,3 +38,68 @@ export function getServiceCountsByStatus(services) {
 
   return { totalRunning, totalStopped };
 }
+
+/**
+ * Check outage status.
+ * 
+ * @param {Array} services 
+ * @returns {Outage}
```

Comment:
`Outage` is not a class or a type function. It's just a numeric data type as we don't have Enum data structures on JavaScript